### ISSUE TITLE
InvalidRequestError: text-babbage-001 is no longer support

### ISSUE THREAD

**Issue by DavidLee528** [leondz/garak#565]
### Exception detail

```bash
Exception has occurred: InvalidRequestError       (note: full exception trace is shown but execution is paused at: _run_module_as_main)
The model `text-babbage-001` has been deprecated, learn more here: https://platform.openai.com/docs/deprecations
  File "/home/tianshu_intel/anaconda3/envs/garak_cpu/lib/python3.11/site-packages/openai/api_requestor.py", line 794, in _interpret_response_line
    raise self.handle_error_response(
  File "/home/tianshu_intel/anaconda3/envs/garak_cpu/lib/python3.11/site-packages/openai/api_requestor.py", line 729, in _interpret_response
    self._interpret_response_line(
  File "/home/tianshu_intel/anaconda3/envs/garak_cpu/lib/python3.11/site-packages/openai/api_requestor.py", line 301, in request
    resp, got_stream = self._interpret_response(result, stream)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tianshu_intel/anaconda3/envs/garak_cpu/lib/python3.11/site-packages/openai/api_resources/abstract/engine_api_resource.py", line 155, in create
    response, _, api_key = requestor.request(
                           ^^^^^^^^^^^^^^^^^^
  File "/home/tianshu_intel/anaconda3/envs/garak_cpu/lib/python3.11/site-packages/openai/api_resources/completion.py", line 25, in create
    return super().create(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tianshu_intel/llm_vulnerability_scanner/garak/garak/generators/openai.py", line 111, in _call_model
    response = self.generator.create(
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tianshu_intel/anaconda3/envs/garak_cpu/lib/python3.11/site-packages/backoff/_sync.py", line 105, in retry
    ret = target(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tianshu_intel/llm_vulnerability_scanner/garak/garak/generators/base.py", line 75, in generate
    return self._call_model(prompt)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tianshu_intel/llm_vulnerability_scanner/garak/garak/probes/base.py", line 96, in _execute_attempt
    this_attempt.outputs = self.generator.generate(this_attempt.prompt)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tianshu_intel/llm_vulnerability_scanner/garak/garak/probes/base.py", line 143, in probe
    attempts_completed.append(self._execute_attempt(this_attempt))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/tianshu_intel/llm_vulnerability_scanner/garak/garak/harnesses/base.py", line 92, in run
    attempt_results = probe.probe(model)
                      ^^^^^^^^^^^^^^^^^^
  File "/home/tianshu_intel/llm_vulnerability_scanner/garak/garak/harnesses/probewise.py", line 108, in run
    h.run(model, [probe], detectors, evaluator, announce_probe=False)
  File "/home/tianshu_intel/llm_vulnerability_scanner/garak/garak/command.py", line 212, in probewise_run
    probewise_h.run(generator, probe_names, evaluator, buffs)
  File "/home/tianshu_intel/llm_vulnerability_scanner/garak/garak/cli.py", line 473, in main
    command.probewise_run(generator, probe_names, evaluator, buffs)
  File "/home/tianshu_intel/llm_vulnerability_scanner/garak/garak/__main__.py", line 13, in main
    cli.main(sys.argv[1:])
  File "/home/tianshu_intel/llm_vulnerability_scanner/garak/garak/__main__.py", line 17, in <module>
    main()
  File "/home/tianshu_intel/anaconda3/envs/garak_cpu/lib/python3.11/runpy.py", line 88, in _run_code
    exec(code, run_globals)
  File "/home/tianshu_intel/anaconda3/envs/garak_cpu/lib/python3.11/runpy.py", line 198, in _run_module_as_main (Current frame)
    return _run_code(code, main_globals, None,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
openai.error.InvalidRequestError: The model `text-babbage-001` has been deprecated, learn more here: https://platform.openai.com/docs/deprecations
```

### Reason Analysis

text-babbage-001 is no longer support by OpenAI, see:

https://platform.openai.com/docs/deprecations#:~:text=2024%2D01%2D04-,text%2Dbabbage%2D001,-%240.0005%20/%201K%20tokens

A Exception would be raise if I follow the test instruction on readme.md

**Comment by DavidLee528** [comment]
The code has been updated in  to solve this deprecation however the readme.md is still using `text-babbage-001` as example.

**Comment by leondz** [comment]
thanks for this, will fix!

**Comment by leondz** [comment]
fixed in #575
